socorro/unittest/processor/test_signature_utilities.py

```diff
@@ -1091,20 +1050,20 @@ def test_action_1(self):
         }
 
         # the call to be tested
-        ok_(sgr._action(raw_crash, {}, processed_crash, processor_meta))
+        assert sgr._action(raw_crash, {}, processed_crash, processor_meta) is True
```

Comment:
For predicate and action returns, I think we really want to verify the actual return value and not whether or not it's truthy, so I made it a test against the actual `True` or `False` value.

That will protect us against accidentally not returning something which in Python will return `None` which is falsey, but possibly a bug.